addusers.py

```diff
@@ -171,6 +171,50 @@ def grant_role(self, auth_url, project_id, user_id, role_id):
         return self.keystone.session.put(url)
 
 
+def manual_testing(header):
```

Comment:
instead of implementing new inputs inside `parse_rows`, I will create inputs to be added as a parameter